rust/relay/src/auth.rs

```diff
@@ -134,32 +134,29 @@ pub(crate) fn systemtime_from_unix(seconds: u64) -> SystemTime {
 mod tests {
     use super::*;
     use crate::Attribute;
-    use hex_literal::hex;
     use stun_codec::rfc5389::attributes::Username;
     use stun_codec::rfc5389::methods::BINDING;
     use stun_codec::{Message, MessageClass, TransactionId};
 
-    const RELAY_SECRET_1: [u8; 32] =
-        hex!("4c98bf59c99b3e467ecd7cf9d6b3e5279645fca59be67bc5bb4af3cf653761ab");
-    const RELAY_SECRET_2: [u8; 32] =
-        hex!("7e35e34801e766a6a29ecb9e22810ea4e3476c2b37bf75882edf94a68b1d9607");
+    const RELAY_SECRET_1: &str = "4c98bf59c99b3e467ecd7cf9d6b3e5279645fca59be67bc5bb4af3cf653761ab";
+    const RELAY_SECRET_2: &str = "7e35e34801e766a6a29ecb9e22810ea4e3476c2b37bf75882edf94a68b1d9607";
     const SAMPLE_USERNAME: &str = "n23JJ2wKKtt30oXi";
 
     #[test]
     fn generate_password_test_vector() {
         let expiry = systemtime_from_unix(60 * 60 * 24 * 365 * 60);
 
-        let password = generate_password(&RELAY_SECRET_1, expiry, SAMPLE_USERNAME);
+        let password = generate_password(RELAY_SECRET_1, expiry, SAMPLE_USERNAME);
 
-        assert_eq!(password, "XnR4dOjSrxVx+3PR5/XIFKA80NckB04N7ndZMM6aoQg")
+        assert_eq!(password, "00hqldgk5xLeKKOB+xls9mHMVtgqzie9DulfgQwMv68")
     }
 
     #[test]
     fn generate_password_test_vector_elixir() {
```

Comment:
Most importantly, this test does not change its assertion which tells us that things still work.